packages/back-end/src/queryRunners/QueryRunner.ts

```diff
@@ -105,9 +105,9 @@ export abstract class QueryRunner<
   private useCache: boolean;
 
   public constructor(
+    context: ReqContext | ApiReqContext,
```

Comment:
Small clean up - we've made the design decision to always ensure `context` is the first prop passed in.